### PR TITLE
remove nbdev

### PR DESCRIPTION
**Pull request by jalew188**
1. Remove all nbdev-related files.
2. Use python -m pytest --nbmake instead of nbdev_test for all nbdev_nbs.

### CODE REVIEW

File: .github/workflows/publish_and_release.yml [hunk 1]
```diff
@@ -25,13 +25,130 @@ jobs:
           cd misc
           . ./check_version.sh
           echo "version=${current_version}" >> $GITHUB_OUTPUT
+  # Create_Draft_On_GitHub:
```
Comment (mschwoer) [inline .github/workflows/publish_and_release.yml:28]: what's the plan with these commented lines?

File: tests/run_tests.sh [hunk 2]
```diff
@@ -1 +1,2 @@
-
+INCLUDED_NBS=$(find ./nbdev_nbs -name "*.ipynb")
```
Comment (mschwoer) [inline tests/run_tests.sh:1]: should we rename `nbdev_nbs` to `nbs` then? 
also, there's still a file `nbdev.yaml` (referenced in `_quarto.yml`) .. could that be deleted?

File: .github/workflows/_run_tests.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - uses: actions/checkout@v3
+    - uses: conda-incubator/setup-miniconda@v3
+      with:
+        miniconda-version: "latest"
+        auto-update-conda: true
+        python-version: ${{ inputs.python-version }}
+    - name: Conda info
+      shell: bash -l {0}
+      run: conda info
+    - name: Perform pip installation with all stable dependencies
```
Comment (mschwoer) [inline .github/workflows/_run_tests.yml:29]: (nit) Perform pip installation with selected (stable/loose) dependencies

**Review by mschwoer — APPROVED** [review]
LGTM!

File: tests/run_tests.sh (hunk 2 above)
Comment (jalew188) [reply tests/run_tests.sh:1]: Will remove

File: .github/workflows/_run_tests.yml (hunk 3 above)
Comment (jalew188) [reply .github/workflows/_run_tests.yml:29]: We can create another PR for this

File: tests/run_tests.sh (hunk 2 above)
Comment (jalew188) [reply tests/run_tests.sh:1]: nbdev_nbs -> nbs_tests

File: .github/workflows/_run_tests.yml (hunk 3 above)
Comment (jalew188) [reply .github/workflows/_run_tests.yml]: Actually, the stable/loose is controlled in pip_intallation.yml, not here

**Pull request merged**